apps/vscode/src/package.json

```
@@ -74,6 +74,16 @@
           "command": "nx.run.fileexplorer",
           "group": "2_workspace"
         },
+        {
+          "when": "isNxWorkspace && config.nxConsole.enableGenerateFromContextMenu",
+          "command": "nx.move.fileexplorer",
+          "group": "2_workspace"
+        },
+        {
+          "when": "!isNxWorkspace && config.nxConsole.enableGenerateFromContextMenu",
```

I think it's better to use `isAngularWorkspace` here. 
```suggestion
          "when": "isAngularWorkspace && config.nxConsole.enableGenerateFromContextMenu",
```